src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.cs

```diff
@@ -0,0 +1,574 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
@hongdai -- could you give us a quick statement of what is new code and what is copied from existing?  It will save time if we can ignore copied code?